### PR TITLE
iz: add support for centos 7

### PR DESCRIPTION
**Pull request by icez**

### CODE REVIEW

**Comment by rlex** [comment]
Maybe use https://docs.ansible.com/ansible/2.5/modules/yum_repository_module.html instead of get_url for repo?

**Comment by icez** [comment]
@rlex i wonder why i miss that module. updated as per suggest =)

File: tasks/install.yml [hunk 1]
```diff
@@ -3,9 +3,21 @@
...
     state: present
     update_cache: yes
+  when: ansible_pkg_mgr == "apt"
+
+- name: Add LizardFS CentOS Repository
+  yum_repository:
+      name: lizardfs
+      description: LizardFS Packages
+      baseurl: "http://packages.lizardfs.com/yum/centos{{ ansible_distribution_major_version|int }}/"
+      gpgcheck: no
```
Comment (rlex) [inline tasks/install.yml]: Signing key is available here: http://packages.lizardfs.com/lizardfs.key
Probably worth using
```
gpgcheck: yes
gpgkey: https://packages.lizardfs.com/lizardfs.key
```

**Comment by icez** [comment]
gpg verify added. thanks

**Comment by rlex** [comment]
Looks great now, thanks!

**Pull request merged**